src/server/views/dashboard/lists-edit.njk

```diff
@@ -18,45 +18,36 @@
     {{ dashboardMacros.successBanner("List updated successfully") }}
   {% endif %}
 
+  {% if annualReviewDateUpdated %}
+    {{ dashboardMacros.successBanner("Updated successfully", "Annual review date updated successfully") }}
+  {% endif %}
+
   {% if error.field %}
     {{ dashboardMacros.errorBanner(error) }}
   {% endif %}
 
+  <div>
+    <h2 class="govuk-heading-m">Annual review</h2>
+    {% if lastAnnualReviewStartDate %}
+      <p class="govuk-body">The last annual review started {{ lastAnnualReviewStartDate }}.</p>
```

Comment:
Another one for Neil.  This sentence seems like it need an "on" or an "at" in it.

Reply:
Yep, good point, I just ripped it straight from the designs but it does read a bit odd 🤔 

<img width="599" alt="Screenshot 2022-11-07 at 16 32 24" src="https://user-images.githubusercontent.com/1377253/200364062-58f20e12-513f-4571-b395-5db1653f121d.png">